pkgs/top-level/all-packages.nix

```diff
@@ -12955,6 +12955,8 @@ in
       withQt5 = true;
     };
 
+    qtutilities = callPackage ../development/libraries/qtutilities { };
```

Comment:
Oh, thanks. I wasn't sure of that at the beginning because I could notice that some packages _are_ defined twice like that, e.g `qtkeychain`. The missing piece for me was the fact that I couldn't find a way to address it with `nix-build -A`. Now I know it's with `libsForQt${version}.qtkeychain`.